webhook_server/libs/handlers/issue_comment_handler.py

```
@@ -161,6 +165,25 @@ async def user_commands(
 
+        # Check if command is allowed on draft PRs
+        if is_draft:
+            allow_commands_on_draft = self.github_webhook.config.get_value("allow-commands-on-draft-prs")
+            # Empty list means all commands allowed; non-empty list means only those commands
+            if isinstance(allow_commands_on_draft, list) and len(allow_commands_on_draft) > 0:
+                # Sanitize: ensure all entries are strings for safe join and comparison
+                allow_commands_on_draft = [str(cmd) for cmd in allow_commands_on_draft]
+                if _command not in allow_commands_on_draft:
+                    self.logger.debug(
+                        f"{self.log_prefix} Command {_command} is not allowed on draft PRs. "
+                        f"Allowed commands: {allow_commands_on_draft}"
```

Ruff (0.15.0)

[warning] Logging statement uses f-string

(G004)